docs/csharp/language-reference/keywords/snippets/abstract/Program.cs

```diff
@@ -0,0 +1,83 @@
+//<snippet1>
+public abstract class Vehicle
+{
+    protected string _brand;
+    
+    // Constructor - implemented method in abstract class
+    public Vehicle(string brand)
+    {
+        _brand = brand;
+    }
```

Comment:
This should also be an expression bodied member:

```suggestion
    public Vehicle(string brand) => _brand = brand;
```